main/vstring.h

```diff
@@ -22,7 +22,7 @@
 *   MACROS
 */
 #ifndef DEBUG
-# define VSTRING_PUTC_MACRO 1
+#define VSTRING_PUTC_MACRO 1
```

Comment:
Everywhere in the code base there are these spaces to indent. See `debug.h` or `general.h` as examples. So please that kind of thing as they are.